app/assets/javascripts/delete.js

```diff
@@ -1,3 +1,21 @@
-$(() => {
+'use strict'
 
+$(() => {
+    deleteIdea();
 })
+
+var deleteIdea = () => {
+    $('#ideas').delegate('#delete-idea', 'click', (event) => {
+        let $idea = $(event.toElement);
+        $.ajax({
+            type: 'DELETE',
+            url: '/api/v1/ideas/' + $idea.attr('idea-id'),
+            success: () => {
+                $idea.parent().remove();
+            },
+            error: (xhr) => {
+                alert(xhr.responseTest)
+            }
+        })
+    })
```

Comment:
This is another great opportunity to use the promise-syntax using `.then()` chained onto the AJAX call.